docs/src/doc/setup/gradle.md

```diff
@@ -0,0 +1,94 @@
+This binding uses [Gradle](https://gradle.org) as its build tool and you will need version 6.0 or higher installed. The next requirement is to have a Godot project (obviously!), if you don't have it yet please create one.
+
+Open a terminal and `cd` to root directory of your Godot project.
+
+## Wrapper
+On this step, we will be setting up a Gradle [wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html). The wrapper will ensure that anyone who wants to build your project from source will use the same gradle version.
+
+```shell
+touch build.gradle.kts gradle.properties settings.gradle.kts
+```
+
+The above command will create three files, which will be empty for now.
+
+```shell
+gradle wrapper --gradle-version=6.5.1
+```
+
+That is it, you have the wrapper installed! The command will produce several files but the important ones are `gradlew` and `gradlew.bat`. Moving forward we will be using `gradlew` to run gradle (`gradlew.bat` on Windows). The first time `gradlew` is used it will download the gradle version you have specified before.
+
+## Setup
+Once you have the wrapper installed, we need to setup the Gradle plugin this binding provides. Without the plugin, you will have to manually generate the entry point, `.gdnlib` and `.gdns` files.
+
+```kotlin
+plugins {
+    kotlin("multiplatform") version "$kotlinVersion"
+    id("com.utopia-rise.godot-kotlin") version "$godotKotlinVersion"
+}
+
+repositories {
+    jcenter()
+    mavenCentral()
+}
+
+godot {
+    // Build a debug binary
+    debug.set(true)
+
+    configureTargets {
+        compilations.getByName("main") {
+            cinterops {
```

Comment:
I would remove this in the initial getting started as its not needed and it might confuse people